src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -1479,7 +1479,7 @@ Namespace Microsoft.CodeAnalysis.Semantics
             If boundNode.Kind = BoundKind.Interpolation Then
                 Return DirectCast(Create(boundNode), IInterpolatedStringContent)
             Else
-                Return CreateBoundInterpolatedStringTextOperation(boundNode)
+                Return CreateBoundInterpolatedStringTextOperation(DirectCast(boundNode, BoundLiteral))
```

Comment:
I didn't understand the purpose of the cast. There seems to only be one `CreateBoundInterpolatedStringTextOperation` overload.